common/beerocks/scripts/prplmesh_utils.sh.in

```diff
@@ -302,17 +302,26 @@ status_function() {
 
     if  pgrep -l beerocks_cont ; then
         if execute_beerocks_command "$bml_cmd" ; then
+            OUTPUT=$output
             # Expecting
             # > OK Main radio agent operational
             # > OK wlan0 radio agent operational
             # > OK wlan2 radio agent operational
-            OK_Count=$(echo "$output" | grep -c -e "OK.*operational")
+            OK_Count=$(echo "$OUTPUT" | grep -c -e "OK.*operational")
             if [ "$OK_Count" -eq 3 ]; then
                 success "operational test success!"
+                echo $OUTPUT | grep -oP '(FAIL| OK).*?operational' | sed -e "s/ OK/OK/"
                 exit 0
             else
                 err "operational test failed!"
-                err "$output"
+                OUTPUT=$(echo $OUTPUT | grep -oP '(FAIL| OK).*?operational' | sed -e "s/ OK/OK/")
+                echo "$OUTPUT"
+                if [ "$(echo "$OUTPUT" | grep -c -e " @BEEROCKS_WLAN1_IFACE@ .*operational")" -eq 0 ]; then
```

Comment:
```suggestion
                if [ "$(echo "$OUTPUT" | grep -c -e " @PRPLMESH_WLAN1_IFACE@ .*operational")" -eq 0 ]; then
```

Reply:
left unchanged, because @PRPLMESH_WLAN1_IFACE@ is not yet defined.